packages/gatsby-plugin-utils/src/polyfill-remote-file/graphql/gatsby-image-resolver.ts

```diff
@@ -261,7 +261,7 @@ export function generateGatsbyImageFieldConfig(
   IGatsbyImageDataArgs
 > {
   return {
-    type: hasFeature(`graphql-typegen`) ? `GatsbyImageData!` : `JSON`,
+    type: hasFeature(`graphql-typegen`) ? `GatsbyImageData` : `JSON`,
```

Comment:
There are some snapshots to updates in `packages/gatsby/src/schema/__tests__/print.js` ( https://app.circleci.com/pipelines/github/gatsbyjs/gatsby/82927/workflows/bde5b2b0-03bb-40b0-9d0b-62f4df912545/jobs/986459 )

Reply:
already done :)